<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>697</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>2</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Монитор ТДО-50 (Система)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QComboBox" name="comboBox_comport">
      <property name="currentText">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>адрес modbus:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QComboBox" name="comboBox_modbus_address">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="pushButton_C1">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Прочитать EEPROM (C1)</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton_C2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Записать EEPROM (C2)</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QPushButton" name="pushButton_C3">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Прочитать состояние (C3)</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="pushButton_C4">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Прочитать заводской номер (C4)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="exchage_image">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="tdo50-resources.qrc">:/icon-gray</pixmap>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="label_serial_number">
      <property name="text">
       <string>Заводской номер:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="4">
     <widget class="QTableWidget" name="tableWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>150</number>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>29</number>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>30</number>
      </attribute>
      <row>
       <property name="text">
        <string>1</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>2</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>3</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>4</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>5</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>6</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>7</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>8</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>9</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>10</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>11</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>12</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>13</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>14</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>15</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>16</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>17</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>18</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>19</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>20</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>21</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>22</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>23</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>24</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>25</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>26</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>27</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>28</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>29</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>30</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>31</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>32</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>33</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>34</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>35</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>36</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>37</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>38</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>39</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>40</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>41</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>42</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>43</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>44</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>45</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>46</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>47</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>48</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>49</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>50</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>Цепь</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Обработка</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Сигнал</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Источник сигнала</string>
       </property>
      </column>
      <item row="0" column="2">
       <property name="text">
        <string>ЭАС</string>
       </property>
      </item>
      <item row="0" column="3">
       <property name="text">
        <string>Автомат 1</string>
       </property>
      </item>
      <item row="1" column="2">
       <property name="text">
        <string>ЭАС</string>
       </property>
      </item>
      <item row="1" column="3">
       <property name="text">
        <string>Автомат 2</string>
       </property>
      </item>
      <item row="2" column="2">
       <property name="text">
        <string>ЭАС</string>
       </property>
      </item>
      <item row="2" column="3">
       <property name="text">
        <string>Автомат 3</string>
       </property>
      </item>
      <item row="3" column="2">
       <property name="text">
        <string>ЭАС</string>
       </property>
      </item>
      <item row="3" column="3">
       <property name="text">
        <string>Автомат 4</string>
       </property>
      </item>
      <item row="4" column="2">
       <property name="text">
        <string>ЭАС</string>
       </property>
      </item>
      <item row="4" column="3">
       <property name="text">
        <string>Автомат 5</string>
       </property>
      </item>
      <item row="5" column="2">
       <property name="text">
        <string>ЭАС</string>
       </property>
      </item>
      <item row="5" column="3">
       <property name="text">
        <string>Автомат 6</string>
       </property>
      </item>
      <item row="6" column="2">
       <property name="text">
        <string>ЭАС</string>
       </property>
      </item>
      <item row="6" column="3">
       <property name="text">
        <string>Автомат 7</string>
       </property>
      </item>
      <item row="7" column="2">
       <property name="text">
        <string>ЭАС</string>
       </property>
      </item>
      <item row="7" column="3">
       <property name="text">
        <string>Автомат 8</string>
       </property>
      </item>
      <item row="8" column="2">
       <property name="text">
        <string>ЭАС</string>
       </property>
      </item>
      <item row="8" column="3">
       <property name="text">
        <string>Автомат 9</string>
       </property>
      </item>
      <item row="9" column="2">
       <property name="text">
        <string>ЭАС</string>
       </property>
      </item>
      <item row="9" column="3">
       <property name="text">
        <string>Автомат 10</string>
       </property>
      </item>
      <item row="10" column="2">
       <property name="text">
        <string>ЭАС</string>
       </property>
      </item>
      <item row="10" column="3">
       <property name="text">
        <string>Питание ВК-4-48</string>
       </property>
      </item>
      <item row="11" column="2">
       <property name="text">
        <string>ЭАС</string>
       </property>
      </item>
      <item row="11" column="3">
       <property name="text">
        <string>ЭАС панели вентиляции</string>
       </property>
      </item>
      <item row="12" column="2">
       <property name="text">
        <string>ЭАС</string>
       </property>
      </item>
      <item row="12" column="3">
       <property name="text">
        <string>Автомат ВК-4-48</string>
       </property>
      </item>
      <item row="13" column="2">
       <property name="text">
        <string>ЭАС</string>
       </property>
      </item>
      <item row="13" column="3">
       <property name="text">
        <string>Автомат ТДО-50</string>
       </property>
      </item>
      <item row="14" column="2">
       <property name="text">
        <string>ЭАС</string>
       </property>
      </item>
      <item row="14" column="3">
       <property name="text">
        <string>Автомат полки ГЧ</string>
       </property>
      </item>
      <item row="15" column="2">
       <property name="text">
        <string>ЭАС</string>
       </property>
      </item>
      <item row="15" column="3">
       <property name="text">
        <string>ЭАС 16</string>
       </property>
      </item>
      <item row="16" column="2">
       <property name="text">
        <string>ЭАС</string>
       </property>
      </item>
      <item row="16" column="3">
       <property name="text">
        <string>ЭАС 17</string>
       </property>
      </item>
      <item row="17" column="2">
       <property name="text">
        <string>ЭАС</string>
       </property>
      </item>
      <item row="17" column="3">
       <property name="text">
        <string>ЭАС 18</string>
       </property>
      </item>
      <item row="18" column="2">
       <property name="text">
        <string>ЭАС</string>
       </property>
      </item>
      <item row="18" column="3">
       <property name="text">
        <string>ЭАС 19</string>
       </property>
      </item>
      <item row="19" column="2">
       <property name="text">
        <string>ЭАС</string>
       </property>
      </item>
      <item row="19" column="3">
       <property name="text">
        <string>ЭАС 20</string>
       </property>
      </item>
      <item row="20" column="2">
       <property name="text">
        <string>ЭАС</string>
       </property>
      </item>
      <item row="20" column="3">
       <property name="text">
        <string>ЭАС 21</string>
       </property>
      </item>
      <item row="21" column="2">
       <property name="text">
        <string>ЭАС</string>
       </property>
      </item>
      <item row="21" column="3">
       <property name="text">
        <string>ЭАС 22</string>
       </property>
      </item>
      <item row="22" column="2">
       <property name="text">
        <string>ЭАС</string>
       </property>
      </item>
      <item row="22" column="3">
       <property name="text">
        <string>ЭАС 23</string>
       </property>
      </item>
      <item row="23" column="2">
       <property name="text">
        <string>ЭАС</string>
       </property>
      </item>
      <item row="23" column="3">
       <property name="text">
        <string>ЭАС 24</string>
       </property>
      </item>
      <item row="24" column="2">
       <property name="text">
        <string>ЭАС</string>
       </property>
      </item>
      <item row="24" column="3">
       <property name="text">
        <string>ЭАС 25</string>
       </property>
      </item>
      <item row="25" column="2">
       <property name="text">
        <string>ЭАС</string>
       </property>
      </item>
      <item row="25" column="3">
       <property name="text">
        <string>ЭАС 26</string>
       </property>
      </item>
      <item row="26" column="2">
       <property name="text">
        <string>ЭАС</string>
       </property>
      </item>
      <item row="26" column="3">
       <property name="text">
        <string>ЭАС 27</string>
       </property>
      </item>
      <item row="27" column="2">
       <property name="text">
        <string>ЭАС</string>
       </property>
      </item>
      <item row="27" column="3">
       <property name="text">
        <string>ЭАС 28</string>
       </property>
      </item>
      <item row="28" column="2">
       <property name="text">
        <string>ЭАС</string>
       </property>
      </item>
      <item row="28" column="3">
       <property name="text">
        <string>ЭАС 29</string>
       </property>
      </item>
      <item row="29" column="2">
       <property name="text">
        <string>ЭАС</string>
       </property>
      </item>
      <item row="29" column="3">
       <property name="text">
        <string>ЭАС 30</string>
       </property>
      </item>
      <item row="30" column="2">
       <property name="text">
        <string>ЭАС</string>
       </property>
      </item>
      <item row="30" column="3">
       <property name="text">
        <string>ЭАС 31</string>
       </property>
      </item>
      <item row="31" column="2">
       <property name="text">
        <string>ЭАС</string>
       </property>
      </item>
      <item row="31" column="3">
       <property name="text">
        <string>ЭАС 32</string>
       </property>
      </item>
      <item row="32" column="2">
       <property name="text">
        <string>НСД</string>
       </property>
      </item>
      <item row="32" column="3">
       <property name="text">
        <string>НСД 33</string>
       </property>
      </item>
      <item row="33" column="2">
       <property name="text">
        <string>НСД</string>
       </property>
      </item>
      <item row="33" column="3">
       <property name="text">
        <string>НСД 34</string>
       </property>
      </item>
      <item row="34" column="2">
       <property name="text">
        <string>НСД</string>
       </property>
      </item>
      <item row="34" column="3">
       <property name="text">
        <string>НСД 35</string>
       </property>
      </item>
      <item row="35" column="2">
       <property name="text">
        <string>НСД</string>
       </property>
      </item>
      <item row="35" column="3">
       <property name="text">
        <string>НСД 36</string>
       </property>
      </item>
      <item row="36" column="2">
       <property name="text">
        <string>НСД</string>
       </property>
      </item>
      <item row="36" column="3">
       <property name="text">
        <string>НСД 37</string>
       </property>
      </item>
      <item row="37" column="2">
       <property name="text">
        <string>НСД</string>
       </property>
      </item>
      <item row="37" column="3">
       <property name="text">
        <string>НСД 38</string>
       </property>
      </item>
      <item row="38" column="2">
       <property name="text">
        <string>НСД</string>
       </property>
      </item>
      <item row="38" column="3">
       <property name="text">
        <string>НСД 39</string>
       </property>
      </item>
      <item row="39" column="2">
       <property name="text">
        <string>НСД</string>
       </property>
      </item>
      <item row="39" column="3">
       <property name="text">
        <string>НСД 40</string>
       </property>
      </item>
      <item row="40" column="2">
       <property name="text">
        <string>НСД</string>
       </property>
      </item>
      <item row="40" column="3">
       <property name="text">
        <string>НСД 41</string>
       </property>
      </item>
      <item row="41" column="2">
       <property name="text">
        <string>НСД</string>
       </property>
      </item>
      <item row="41" column="3">
       <property name="text">
        <string>НСД 42</string>
       </property>
      </item>
      <item row="42" column="2">
       <property name="text">
        <string>НСД</string>
       </property>
      </item>
      <item row="42" column="3">
       <property name="text">
        <string>НСД 43</string>
       </property>
      </item>
      <item row="43" column="2">
       <property name="text">
        <string>НСД</string>
       </property>
      </item>
      <item row="43" column="3">
       <property name="text">
        <string>НСД 44</string>
       </property>
      </item>
      <item row="44" column="2">
       <property name="text">
        <string>НСД</string>
       </property>
      </item>
      <item row="44" column="3">
       <property name="text">
        <string>НСД 45</string>
       </property>
      </item>
      <item row="45" column="2">
       <property name="text">
        <string>НСД</string>
       </property>
      </item>
      <item row="45" column="3">
       <property name="text">
        <string>НСД 46</string>
       </property>
      </item>
      <item row="46" column="2">
       <property name="text">
        <string>НСД</string>
       </property>
      </item>
      <item row="46" column="3">
       <property name="text">
        <string>НСД 47</string>
       </property>
      </item>
      <item row="47" column="2">
       <property name="text">
        <string>НСД</string>
       </property>
      </item>
      <item row="47" column="3">
       <property name="text">
        <string>НСД 48</string>
       </property>
      </item>
      <item row="48" column="2">
       <property name="text">
        <string>НСД</string>
       </property>
      </item>
      <item row="48" column="3">
       <property name="text">
        <string>НСД 49</string>
       </property>
      </item>
      <item row="49" column="2">
       <property name="text">
        <string>НСД</string>
       </property>
      </item>
      <item row="49" column="3">
       <property name="text">
        <string>НСД шкафа</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="tdo50-resources.qrc"/>
 </resources>
 <connections/>
</ui>
